impl/c-qube/src/services/csv-adapter/parser/utils/csvreader.spec.ts

```diff
@@ -37,4 +41,38 @@ describe('CSVReader', () => {
   test('benchmarking', () => {
     // add a benchmark test here similar to date parser
   });
-});
+
+  test('parse with different CSV delimiter', async () => {
+    const res = await readCSV(
+      './test/fixtures/test-csvs/csvreader/invalid.reader.csv'
+    );
+
+    expect(res).toBeDefined();
```

Comment:
don't only check the result to be defined, since you are parsing with a different delimiter, make sure it is parsed correctly.